src/NUnitFramework/tests/Assertions/AssertThrowsTests.cs

```diff
@@ -169,6 +169,7 @@ public void BaseExceptionThrown()
         }
 
         [Test]
+        [SetUICulture("en-US")]
         public void DerivedExceptionThrown()
```

Comment:
Failure reason: Test asserts a constant English exception message that doesn't match the localized exception message.